run-test

```diff
@@ -1,13 +1,35 @@
 #!/bin/bash -e
+# For coverage tests, run as
+#     coverage erase
+#     PATH=$PWD:$PATH RUNNER="coverage run --include=$PWD/**/*" ./run-test
+#     coverage html
+# See ./run-coverage
+RUNNER="${RUNNER:-python2}"
```

Comment:
This is the binary that the scripts are executed by. It defaults to `python2` if not set in the environment.